getting-started/README.md

```diff
@@ -0,0 +1,77 @@
+# Getting Started with OPEA
+
+## Prerequisites
+
+To get started with OPEA you need the right hardware and basic software setup.
+
+Hardware Requirements: For the hardware configuration, If you need Hardware Access visit the Intel Tiber Developer Cloud to select from options such as Xeon or Gaudi processors that meet the necessary specifications.
+Software Requirements: Please refer to the Support Matrix[ Hyper link needed]  to ensure you have the required software components in place.
+
+## Understanding OPEA's Core Components
+
+Before moving forward, it's important to familiarize yourself with two key elements of OPEA: GenAIComps and GenAIExamples.
+1.	GenAIComps: GenAIComps is a collection of microservice components that form a service-based toolkit. This includes a variety of services such as llm (language learning models), embedding, and reranking, among others.
+2.	GenAIExamples: While GenAIComps offers a range of microservices, GenAIExamples provides practical, deployable solutions to help users implement these services effectively. Examples include ChatQnA and DocSum, which leverage the microservices for specific applications. 
+
+## Visual Guide to Deployment
+To illustrate, here's a simplified visual guide on deploying a ChatQnA GenAIExample, showcasing how you can set up this solution in just a few steps. 
+
+![Getting started with OPEA](assets/getting_started.gif)
+
+## Setup ChatQna Parameters
+To deploy ChatQnA services, follow these steps:
+
+### Set the required environment variables:
+```
+    # Example: host_ip="192.168.1.1"

```

Comment:
Don't indent and of these lines in the codeblock (it adds an extra indent in the output that looks odd)